server/config.ts

```diff
@@ -130,6 +132,16 @@ export default {
       agent: new AgentConfig(Number(get('ORCHESTRATION_API_TIMEOUT_RESPONSE', 10000))),
     },
   },
+  features: {
+    notificationTypes: {
+      enabledNotifications: <NotificationType[]>(
+        get(
+          'FEATURE_ENABLED_NOTIFICATION_TYPES',
+          'PRISONER_RELEASED_EVENT,PRISONER_RECEIVED_EVENT,PRISON_VISITS_BLOCKED_FOR_DATE',
```

Comment:
Just have this default config in for now, but could enable additional types via Helm config - e.g. for testing in dev environment.